app/_src/guides/progressively-rolling-out-unified-resource-naming.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Progressively rolling out unified resource naming
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-system = site.mesh_namespace | default: "kuma-system" %}
+
+By default, Envoy resources and stats in {{ Kuma }} use mixed, legacy formats. Names often do not line up with the {{ Kuma }} resources that produced them, which makes dashboards noisy and troubleshooting slower. For example, a query like:
+
+```text
+sum:envoy.cluster.upstream_rq.count{service:my-example-service, !envoy_cluster:kuma_*, !envoy_cluster:meshtrace_*, !envoy_cluster:access_log_sink} by {envoy_cluster}.as_count()
+```
+
+is not intuitive and does not point cleanly back to the right {{ Kuma }} resource. Different resources and their related stats often look unrelated, even when they describe the same traffic path.
+
+Starting with 2.12, you can adopt a unified resource naming scheme that makes names predictable, consistent, and directly tied to {{ Kuma }} resources. This improves observability, simplifies queries, and makes it much easier to understand what is happening in the mesh.
+
+## Prerequisites
+
+You have completed the [Kubernetes Quickstart]({{ docs }}/quickstart/kubernetes-demo/) and have the demo environment running. This guide assumes you already have the `demo-app` service running from the quickstart.
+
+<!-- vale Google.Headings = NO -->
+## Step 1: Create a ContainerPatch
+<!-- vale Google.Headings = YES -->
+
+Create and apply a `ContainerPatch` resource that enables unified naming on the sidecar:
+
+```yaml
+# containerpatch-unified-naming.yaml
+apiVersion: kuma.io/v1alpha1
```

Comment:
can we do this with `echo " " | kubectl apply -f -` like we do in all other guides?

Reply:
addressed